persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java

```diff
@@ -77,24 +78,29 @@ private TomlSyntaxGenerator() {
      * Method to create a new Config.toml file with database configurations.
      */
 
-    public static SyntaxTree createConfigToml(List<String> schemas, String packageName) {
+    public static String createConfigToml(List<Path> schemas, String packageName) {
         NodeList<DocumentMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createEmptyNodeList();
-        for (String schema : schemas) {
-            if (!schema.equals(packageName)) {
-                moduleMembers = moduleMembers.add(SampleNodeGenerator.createTable(packageName + "." + schema, null));
-            } else {
-                moduleMembers = moduleMembers.add(SampleNodeGenerator.createTable(packageName, null));
+        for (Path schemaPath : schemas) {
+            Path schemaFile = schemaPath.getFileName();
+            if (Objects.nonNull(schemaFile)) {
+                String schema = schemaFile.toString().replaceAll(".bal", "");
+                if (!schema.equals(packageName)) {
+                    moduleMembers = moduleMembers.add(SampleNodeGenerator.createTable(packageName +
+                            "." + schema, null));
```

Comment:
module name should read from the Ballerina.toml file. not the schema file name

Reply:
updated in [566e36d](https://github.com/ballerina-platform/persist-tools/pull/148/commits/566e36dd84d453026fbf2506266f34668c08e1af)